certbot/certbot/_internal/cli/__init__.py

```diff
@@ -171,13 +171,9 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
         ["register", "automation"], "--register-unsafely-without-email", action="store_true",
         default=flag_default("register_unsafely_without_email"),
         help="Specifying this flag enables registering an account with no "
-             "email address. This is strongly discouraged, because in the "
-             "event of key loss or account compromise you will irrevocably "
-             "lose access to your account. You will also be unable to receive "
-             "notice about impending expiration or revocation of your "
-             "certificates. Updates to the Subscriber Agreement will still "
```

Comment:
Are you sure we can remove this? Should we maybe double check with a lawyer?

Reply:
@jsha, you and Peter added this text 5 years ago. Do you happen to know if we need legal review on this change?

Reply:
I don't think it needs legal review - the subscriber agreement is pretty clear on the topic, and Brad's right that Certbot isn't a Let's Encrypt specific tool, so isn't in charge of communicating aspects of the SA.

While we're in here, maybe we should add something about "notice of problems with your Certbot install that will lead to failure to renew."

Overall I do think the scariness of `--register-unsafely-without-email` has been a huge win for us - Certbot users typically have email addresses, which has definitely helped us reach people when we need to. Other clients often don't.

Reply:
Thanks for the info jsha.

I think your suggestion for expanding this message is a good one and I made that change.